extra/nrnivmodl_core_makefile.in

```diff
@@ -49,8 +55,8 @@ INCLUDES += $(if @reportinglib_INCLUDE_DIR@, -I$(subst ;, -I,@reportinglib_INCLU
 
 # C++ compilation and link commands
 CXX = @CMAKE_CXX_COMPILER@
-CXXFLAGS = @BUILD_TYPE_CXX_FLAGS@ @CMAKE_CXX_FLAGS@ @CXX14_STANDARD_COMPILE_OPTION@ @PGI_INLINE_FLAGS@
```

Comment:
If you remove `CXX14_STANDARD_COMPILE_OPTION` means that we can remove the cmake file to handle c++ versions?

Reply:
Instead of using separate CMake variable, I am adding that flag to CXX_FLAGS. So we will need that handling for PGI compiler.